pkgs/development/python-modules/evohome-async/default.nix

```diff
@@ -1,45 +1,62 @@
 {
   lib,
+  aiofiles,
   aiohttp,
+  aioresponses,
+  aiozoneinfo,
+  asyncclick,
   buildPythonPackage,
-  click,
   fetchFromGitHub,
   hatchling,
+  pytest-asyncio,
+  pytest-freezer,
+  pytestCheckHook,
   pythonOlder,
+  pyyaml,
+  syrupy,
   voluptuous,
 }:
 
 buildPythonPackage rec {
   pname = "evohome-async";
-  version = "0.4.21";
+  version = "1.0.2";
   pyproject = true;
 
-  disabled = pythonOlder "3.11";
+  disabled = pythonOlder "3.12";
 
   src = fetchFromGitHub {
     owner = "zxdavb";
     repo = "evohome-async";
     tag = version;
-    hash = "sha256-+tBqyg7E9wRC6oHq6Fv2vzIGtrHqZp8w/3x/xbacqWI=";
+    hash = "sha256-CpN0QAlUqCDy6hNkuNvbjQUee40BA0UqAleR+Omm9bA=";
   };
 
-  nativeBuildInputs = [ hatchling ];
+  build-system = [ hatchling ];
 
-  propagatedBuildInputs = [
+  dependencies = [
+    aiofiles
     aiohttp
-    click
+    aiozoneinfo
+    asyncclick
     voluptuous
```

Comment:
What about debugpy?
https://github.com/zxdavb/evohome-async/blob/1.0.2/requirements_cli.txt#L18

Reply:
Can we make those optional instead? The upstream does not propagate them either.